src/main/java/com/whytowait/domain/models/MenuCategory.java

```diff
@@ -15,14 +13,37 @@
 @Entity
 @AllArgsConstructor
 @NoArgsConstructor
-
+@Table(name = "menu_category")
 public class MenuCategory {
+
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    private UUID Id;
+    @GeneratedValue
+    private UUID id;
+
+    @NotNull
+    @Column(nullable = false)
+    private String name;
+
+    @NotNull
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "merchant_id", referencedColumnName = "id", nullable = false)
```

Comment:
Now here for every Category, it would keep fetching entire merchant info unnecessary. we can have an api call to fetch Merchant and have just an ID here.

this is okay if we are doing this while fetching merchant that we get entire info fetched but not here i think